stdlib/ballerina-http/src/main/ballerina/swagger/Package.md

```diff
@@ -0,0 +1,86 @@
+## Package overview
+This package provides the following code generation capabilities to Ballerina:
+1. Generate the Ballerina code for a given Swagger definition.
+2. Generate the client stub for an existing Ballerina service at build time.
+3. Export the Swagger definition of a Ballerina service.
+
+The `swagger` command in Ballerina is used for Swagger to Ballerina and Ballerina to Swagger code generation.
+Code generation from Swagger to Ballerina can produce `mock services` and `client stubs`.
+
+For build time client stub generation, annotation support is provided.
+
+### Mock Service from Swagger
+`ballerina swagger mock <swaggerfile> 
+    [-p packagename>|--package <packagename>] 
+    [-o <path>|--output<path>]`
+
+Generates a Ballerina service for the Swagger file.
+
+This generated service is a mock version of the actual service. Generated sources contain the service definition in `gen/` and the resource implementation file in the package root directory with the suffix `_impl`. The `_impl` file is not overwritten by code regeneration.
+
+### Client Stub from Swagger
+`ballerina swagger client <swaggerfile> 
+    [-p packagename>|--package <packagename>] 
+    [-o <path>|--output<path>]`
+    
+Generates a Ballerina client stub for the service defined in a Swagger file.
+
+This client can be used in client applications to call the service defined in the Swagger file.
+
+### Service to Swagger Export
+`ballerina swagger export <balfile> 
+    [-o <path>|--output<path>]
+    [-s <servicename>|--service <servicename>]`
+
+Export the Ballerina service to a definition of OpenApi Specification 3.0.
+For the export to work properly, the input ballerina service should be defined using basic service and resource level HTTP annotations.
```

Comment:
Change "ballerina" to "Ballerina"

Reply:
@appellj Fixed all comments with 66bff6aeb27c4659eaaa6f9c192e47eccb697f75